tests/components/alarm_control_panel/test_device_action.py

```diff
@@ -80,14 +81,14 @@
     ],
 )
 async def test_get_actions(
-    hass,
-    device_registry,
-    entity_registry,
-    set_state,
-    features_reg,
-    features_state,
-    expected_action_types,
-):
+    hass: HomeAssistant,
+    device_registry: dr.DeviceRegistry,
+    entity_registry: er.EntityRegistry,
+    set_state: bool,
+    features_reg: const.AlarmControlPanelEntityFeature,
+    features_state: const.AlarmControlPanelEntityFeature,
```

Comment:
The should have been imported from `homeassistant.component.alarm_control_panel` not `const`.